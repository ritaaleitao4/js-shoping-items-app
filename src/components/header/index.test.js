import Header from '.';

const mockItem = [{
    modelClass: "A",
    version: "200",
    price: {
        amount: "29500.00",
        currency: "EUR",
        locale: "de-DE"
    },
    imagePath: "/assets/carImages/aclasswhite.jpg"
}, {
    modelClass: "B",
    version: "300",
    price: {
        amount: "9500.00",
        currency: "EUR",
        locale: "de-DE"
    },
    imagePath: "/assets/carImages/aclasswhite.jpg"
}];

describe('header component', () => {
    it('it renders correctly', () => {
        expect(new Header(mockItem).render()).toMatchSnapshot();
    });
});