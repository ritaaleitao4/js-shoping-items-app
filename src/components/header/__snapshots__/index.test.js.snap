// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component it renders correctly 1`] = `
<header class="main-header">
  <section>
    <img class="main-header__logo"
         src="[object Object]"
         alt="logo-img"
    >
    <aside class="main-header__bag">
      <p>
        â‚¬39,000.00
      </p>
      <img src="[object Object]"
           alt="bag-icon"
      >
    </aside>
  </section>
</header>
`;
